Core/VariantManager.swift

```diff
@@ -61,6 +64,9 @@ public struct Variant {
         Variant(name: "sd", weight: doNotAllocate, isIncluded: When.always, features: []),
         Variant(name: "se", weight: doNotAllocate, isIncluded: When.inRequiredCountry, features: []),
         
+        Variant(name: "mc", weight: 1, isIncluded: When.always, features: []),
+        Variant(name: "mf", weight: 1, isIncluded: When.always, features: [.fireButtonEducation]),
+
         Variant(name: "mi", weight: 1, isIncluded: When.always, features: []),
         Variant(name: "mo", weight: 1, isIncluded: When.always, features: [.removeSERPHeader])
```

Comment:
Bartek and I have talked about this and we'll probably just run this experiment once his is done